spec/lib/appsignal_spec.rb

```diff
@@ -92,18 +92,23 @@
 
       context "when allocation tracking and gc instrumentation have been enabled" do
         before do
+          GC::Profiler.stub(:enable)
```

Comment:
I did this on purpose, because I'm not expecting or allowing here, but I'm explicitly stubbing it. Also, the rest of the test suite uses `stub()` in this case, except for a couple of cases. We should settle on a preferred style here, and I'm mostly fine with both. wdyt @matsimitsu and @thijsc?